### ISSUE TITLE
utility of term 'asymmetric protein localization'

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#14242]
What is the utility of this term?

@pgaudet  In https://github.com/geneontology/go-ontology/issues/14233, you specifically mentioned this term 

> "GO:1903361 protein localization to basolateral plasma membrane
> should be merged under  GO:0061467 basolateral protein localization, since this term is correctly placed under asymetrical localization."

but there are very few things under it, and the majority of the things under just plain 'protein localization' seem like they would fit the definition of being asymmetrically localized also.

Would people ask for a 'protein localization' term that did NOT mean asymmetric localization such that there is a distinct concentration of the gp of interest in a specific location?

Basically, this term does not seem to be providing a useful distinction between the terms under it specifically and those terms just under 'protein localization'

**Comment by pgaudet** [comment]
@pmasson55 any thoughts on this ?

**Comment by ValWood** [comment]
I don't know if this parent term is required, but  there is a term under here   which is used for asymettric localization with a very specific meaning.

GO:0061509 - asymmetric protein localization to old mitotic spindle pole body

Definition | Parents
-- | --
Any process in which a protein is transported to, or maintained to the old mitotic spindle pole body resulting in its being distributed asymmetrically.

This is to capture that  cells can distinguish between an old and a new spindle pole body which seems to be important for  developmental processes,  and possibly ageing IIRC.

**Comment by krchristie** [comment]
@ValWood - I have no objection to any of the specific terms, just to the grouping term '**asymmetric protein localization**' since it does not seem to be providing a useful distinction between the small number of terms under it and the large number of terms that are just under '**protein localization**'. 

This came up because in https://github.com/geneontology/go-ontology/issues/14233, @pgaudet made the point that 'asymmetric protein localization' was the preferred parentage for the merge of '**basolateral protein localization**' and '**protein localization to basolateral plasma membrane**'. However, another term that I have a lot of familiarity with, '**protein localization to cilium**' is NOT under the term '**asymmetric protein localization**'. How is specific localization to the the cilium more or less asymmetric than specific localization to the basolateral membrane? It does not look to me like there is something different going on with respect to the asymmetric-ness of localization for most of the terms under 'asymmetric protein localization' versus the terms not under it.

Also, only the small handfull of terms that have '**asymmetric**' in the term name mention asymmetric in the definition. And looking at definitions, e.g. for '**asymmetric protein localization to old or new spindle pole body**':

> Any process in which a protein is transported to, or maintained to either the old or new spindle pole body resulting in its being distributed asymmetrically.

I think it would help if we added 'between the old and new spindle pole bodies' after the word 'asymmetrically' for this term and its two child terms to indicate what the asymmetry is between.

I don't know how to help the def for 'asymmetric protein localization involved in cell fate determination'. It just says:

> Any process in which a protein is transported to, or maintained in, a specific asymmetric distribution, resulting in the formation of daughter cells of different types. 

without giving any indication of what the localization is. Hard to see how this is different than just some sort of specific protein localization.

**Comment by ValWood** [comment]
Hi @krchristie  OK yes I agree with you.
Val

**Comment by krchristie** [comment]
To be clear, I think we could delete this grouping term such that all of its current children are just directly under 'protein localization' without losing anything useful.

**Comment by pgaudet** [comment]
Perfect. Thanks !!

**Comment by pgaudet** [comment]
Hi @ValWood 

Thinking about this one further, doesn't protein localization imply some asymmetry ? 
ie I cannot find just ' protein localization to new mitotic spindle pole body' - so the words 'asymmetric' seem unnecessary for these terms. Just wondering. 

Thanks, Pascale

**Comment by pgaudet** [comment]
I am attaching the direct EXP annotations to 'asymmetric protein localization' (for verification and for reference). There are 65 annotations affecting 


UniProt
--
WB @vanaukenk 
FlyBase @hattrill 
SGD @srengel 
PINC @pgaudet 
CGD ????
MGI @ukemi 


I removed the PINC annotation because the annotated protein was the *target* of the asymmetric localization (UniProtKB:Q9Y215 | Acetylcholinesterase collagenic tail peptide | PMID:9689136)

- Pascale

**Comment by hattrill** [comment]
I cannot look at this for a bit. But I think there is a strong case to make for specifying the "asymmetric" part especially with relation to processes that underpin cell polarization.

**Comment by krchristie** [comment]
@ukemi - I can check the MGI ones.

**Comment by srengel** [comment]
SGD has 3 annotations to 2 genes.  i'll review them...

**Comment by srengel** [comment]
sorry, accidentally hit 'close and comment' instead of 'comment'!  oops.

**Comment by pgaudet** [comment]
Hi, 

I agree with you @ukemi - however what we have right now is limited to: 
'asymmetric protein localization involved in cell fate determination'
'asymmetric protein localization to new mitotic spindle pole body' 
'asymmetric protein localization to old mitotic spindle pole body' 
'asymmetric protein localization to old or new spindle pole body' 

I was asking if the 'asymmetric' part was really useful (it may be!). 

Also, we have these other children of 'asymmetric protein localization': 
'apical protein localization'
'basal protein localization'
'inheritance of oxidatively modified proteins involved in replicative cell aging'
'pole plasm protein localization'
'protein localization involved in acrosome reaction'
'protein localization to axon'
'protein localization to myelin sheath abaxonal region'
'protein localization to somatodendritic compartment'

If we remove the parent 'asymmetric protein localization', should we change the labels to include 'asymmetric' somehow'? 

(I am happy to simply just merge 'asymmetric protein localization' with its parent 'protein localization' and leave the other terms as they are.) 

Thanks, Pascale

**Comment by vanaukenk** [comment]
WB has 24 annotations to 18 genes.  I'll review these.  It might be informative to construct a GO-CAM model or two here to see how these existing annotations would fit into a model of the overall process being studied.

**Comment by krchristie** [comment]
Part of the reason I made this ticket is that the terms Pascale listed do not seem to have any greater or lesser 'asymmetry' in their localization that many of the terms that are directly under 'protein localization' but NOT under 'asymmetric protein localization'

> Also, we have these other children of 'asymmetric protein localization':
> 'apical protein localization'
> 'basal protein localization'
> 'inheritance of oxidatively modified proteins involved in replicative cell aging'
> 'pole plasm protein localization'
> 'protein localization involved in acrosome reaction'
> 'protein localization to axon'
> 'protein localization to myelin sheath abaxonal region'
> 'protein localization to somatodendritic compartment'

I've also now checked the three MGI annotations:
- Dync2h1 - moved to 'protein localization to cilium' [coincidentally the exact term I mentioned originally as an example of something where the localization is just as asymmetric as some of the things under 'asymmetric protein localization' though this term is not]
- Scrib - involved in 'localization to basolateral membrane' [**this paper used the phrase 'asymmetric localization a lot**]
- Wnt7a - involved in localization of Vangl2 to opposite poles of daughter cells [no specific term to represent localization to this destination]

Having spent some time today thinking about this, I do not think that there is a different process for 'protein localization' versus 'asymmetric protein localization'. ALL of it is protein localization resulting in an asymmetric distribution, whether it is common for a given field to use the word 'asymmetric' or not. Even for David's example of protein localization to the nucleus, this results in an asymmetric distribution where there is more of the protein in the nucleus than in other places.

The apical/basolateral targeting field is a particularly good case to discuss. The paper for the annotation of the mouse Scrib protein uses the phrase "asymmetric localization" a lot. However, for various reasons, I've read a number of papers and reviews on apical/basolateral targeting/localization lately, and most of them do not use the word 'asymmetric', but just discuss targeting, hence why I was surprised to find the term 'basolateral protein localization' under 'aysmmetric protein localization'.

I don't have any objection to term names that include the word 'asymmetric' if that's standard for a given area, like the term @ValWood brought up: **GO:0061509 - asymmetric protein localization to old mitotic spindle pole body**. 

However, if we went through all the child terms of 'protein localization' that are NOT currently under 'asymmetric protein localization', how many of them would NOT fit the definition for 'asymmetric protein localization':

> Any process in which a protein is transported to, or maintained in, a specific location such that it is distributed asymmetrically.

**Comment by vanaukenk** [comment]
Looking over the WB annotations, I think that a number of these could be annotated to the more specific term of 'asymmetric protein localization involved in cell fate determination', so if we keep that term, I could transfer our annotations and they would still be okay.

**Comment by srengel** [comment]
i removed the 3 SGD annotations.  2 were old and questionable anyway.  the 3rd was covered by other annotations from the same paper (cellular bud site selection and actin cortical patch localization).

**Comment by pgaudet** [comment]
Hi Kimberly, 

'asymmetric protein localization involved in cell fate determination' will remain, yes. 

Pascale
